packages/gallery/src/components/styles/animations.scss

```diff
@@ -0,0 +1,5 @@
+:global {
+    .animation-slide {
+        transition: all 0.4s ease
```

Comment:
this is very bad css.
Use the actual properties you need to animate - never use `all`